python_tse/rdv_resumo.py

```diff
@@ -28,7 +28,8 @@ class Fase(asn1.Enumerated):
     _map = {
         1: "simulado",
         2: "oficial",
-        3: "treinamento",
+        3: "treinamento",if voto13t
```

Comment:
```suggestion
        3: "treinamento", if voto13t
```
Por favor, adicione um espaço depois da virgula. 
Depois que você testar com "treinamento", poderia fazer o mesmo nas eleiçoes de 2026?